spring-batch/src/test/java/org/baeldung/conditionalflow/step/NumberInfoGeneratorUnitTest.java

```diff
@@ -0,0 +1,20 @@
+package org.baeldung.conditionalflow.step;
+
+import org.baeldung.conditionalflow.model.NumberInfo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class NumberInfoGeneratorUnitTest {
+    @Test
+    public void testGenerateNumbers() {
```

Comment:
The same comment applies for test names. And the formatting is off in this class.

Reply:
Done